src/main/java/ua/edu/internship/interview/service/business/InterviewService.java

```diff
@@ -0,0 +1,152 @@
+    }
+
+    public InterviewDto updateInterview(String id, InterviewUpdateDto dto) {
+        log.info("Attempting to update interview with id: {}", id);
+        InterviewDocument interviewDocument = getInterviewByIdOrElseThrow(id);
+        interviewMapper.updateDocument(dto, interviewDocument);
+        validateInterviewNotCausingConflicts(interviewDocument);
+        InterviewDocument updatedInterview = interviewRepository.save(interviewDocument);
+        log.info("Updated interview with id: {}", updatedInterview.getId());
+        return interviewMapper.toDto(updatedInterview);
+    }
+
+    public void deleteInterviewById(String interviewId) {
+        log.info("Attempting to delete interview with id: {}", interviewId);
+        interviewRepository.deleteById(interviewId);
+        log.info("Interview with id: {} deleted successfully", interviewId);
+    }
+
+    public InterviewDto updateInterviewStatus(String interviewId, InterviewStatus newStatus) {
+        log.info("Attempting to update status for interview with id: {}", interviewId);
+        InterviewDocument interviewDocument = getInterviewByIdOrElseThrow(interviewId);
+        InterviewStatus.validateStatusTransition(interviewDocument.getStatus(), newStatus);
+        interviewDocument.setStatus(newStatus);
+        InterviewDocument updatedInterview = interviewRepository.save(interviewDocument);
+        log.info("Updated status for interview with id: {}", updatedInterview.getId());
+        return interviewMapper.toDto(updatedInterview);
+    }
+
+    public InterviewDto updateInterviewFeedback(String interviewId, String feedback) {
+        InterviewDocument interviewDocument = getInterviewByIdOrElseThrow(interviewId);
+        interviewDocument.setFeedback(feedback);
+        InterviewDocument updatedInterview = interviewRepository.save(interviewDocument);
+        log.info("Updated feedback for interview with id: {}", updatedInterview.getId());
+        return interviewMapper.toDto(updatedInterview);
+    }
+
+    public InterviewQuestionDto createInterviewQuestion(String interviewId, InterviewQuestionCreateDto dto) {
+        log.info("Attempting to create interview question for interview with id: {}", interviewId);
+        InterviewQuestionDocument questionDocument = questionMapper.toDocument(dto);
+        UserQuestionDocument userQuestionDocument = getUserQuestionByIdOrElseThrow(dto.getUserQuestionId());
+        questionDocument.setQuestion(userQuestionDocument);
+
+        InterviewDocument interviewDocument = getInterviewByIdOrElseThrow(interviewId);
+        interviewDocument.addQuestion(questionDocument);
+        interviewRepository.save(interviewDocument);
+        log.info("Created interview question for interview with id: {}", interviewDocument.getId());
+        return questionMapper.toDto(questionDocument);
+    }
+
+    public InterviewQuestionDto updateInterviewQuestion(String interviewId, String questionId, InterviewQuestionUpdateDto dto) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 120 characters (found 128).
<!-- __reviewdog__:ChBlNTcyY2JiYzJiOWJhN2RmEgpjaGVja3N0eWxl -->